### PR TITLE
[WIP] Move more common test schedule parts into common schedule functions

### PR DESCRIPTION
**Pull request by okurz**
Verified with local schedule validation on various job settings. I am sure I
missed something :)

### CODE REVIEW

File: lib/main_common.pm [hunk 1]
```diff
@@ -1717,6 +1751,21 @@ sub load_ssh_key_import_tests {
     loadtest "x11/ssh_key_verify";
 }
 
+sub load_common_opensuse_sle_tests {
+    load_autoyast_clone_tests           if get_var('CLONE_SYSTEM');
```
Comment (asmorodskyi) [inline lib/main_common.pm]: I think you changing model here in dangerous way - before we had elsif chain , so if you get into one branch you don't have chance to get into another , with your approach we will be capable to  execute wicked and systemd testsuite in single job which not always make sense

**Comment by rwx788** [comment]
Let's wait for more reviews, but I fully support this idea. After merge we can check what we've missed.

File: products/sle/main.pm [hunk 2]
```diff
@@ -984,8 +933,10 @@ elsif (get_var('HPC')) {
         }
     }
 }
-elsif (get_var('SYSTEMD_TESTSUITE')) {
-    load_systemd_patches_tests;
+elsif (get_var("SES5_DEPLOY")) {
```
Comment (asmorodskyi) [inline products/sle/main.pm]: it's look like something unrelated to this refactoring, right ?

Comment (asmorodskyi) [reply products/sle/main.pm]: if so , please put it at least in separate commit

**Comment by okurz** [comment]
I guess I would need to check the schedule evaluation based on more vars.json files locally

File: lib/main_common.pm (hunk 1 above)
Comment (asmorodskyi) [reply lib/main_common.pm]: other way would be just put as prefix `return` :
`load_toolchain_tests                if get_var('TCM') || check_var('ADDONS', 'tcm');`

**Pull request closed without merging**